app/app.js

```diff
@@ -31,7 +31,6 @@ function start() {
 
 function initialSetup() {
   if ($('.js-diff-progressive-spinner').length || !$('#files').length) {
-    setTimeout(initialSetup, 100);
```

Comment:
This is not needed anymore, as the `setInterval` of `start()` will recall this when the tree is missing.